### DISCUSSION TITLE
OSC2023 東京秋（10/21 リアル展示）へのブース出展について

### DISCUSSION THREAD

**Discussion by potappo** [mozilla-japan/translation#711] [General]
前回 #678 
OSC2023 東京秋の日程が 2023年10月21日(土) 10:00～16:00 で確定しました。8 月上旬から出展者募集予定とのことです。

場所は、[大田区産業プラザPiO](https://www.pio-ota.net/) 大展示ホールとなります。駅は京浜急行とJRの蒲田駅（それぞれ若干距離がある別の駅です）が最寄です。

私的には、仕事の忙しさが10月には落ち着く見込みなので、春同様、私の方で出展参加申込をする予定です。

ブース出展へ協力していただける方を春同様に募集します。全時間帯でなくてもかまいません。昼食や他ブースへの訪問も考えて、交代制でブース出展の方は実施します。

日程を見て気づいた方もいるかと思いますが、10月の翻訳ミートアップの開催予定日付と被っています。
展示会場とオンラインで並行開催も難しいかと思いますので、1 週前（10/14) か 1 週後 (10/28) にずらす予定です。

どっちでやるか悩み中なので、翻訳ミートアップ日程については、申込募集ではなく仮集計ということで、調整さんを別途立てます。

**Comment by potappo** [comment]
ミートアップの調整さんを作りました。
https://chouseisan.com/s?h=f248c978401c476c88b34c0bef1a553c

**Comment by hmatrjp** [comment]
メモ：このOSCのオンラインセミナーは、10/29開催で、ブースとは日が違う

**Reply by gurezo** [reply]
オンライン：2023年9月29日(金)-30日(土) 10:00～18:00
オフライン：2023年10月21日(土) 10:00～16:00

**Reply by potappo** [reply]
オンラインの申込締切は明日です。今日、明日と私は仕事忙しいので、そこまで手が回りません。

オンラインの方は東京秋にこだわらず、継続して参加できるタイミングを探すのが良さそうかと思います。

リアル展示の方は今日の夜か明日の朝に申込書を提出予定です。

**Reply by gurezo** [reply]
そうしましたら、オフラインだけにしましょう。
申込の手続きの方、よろしくお願い申し上げます。

**Comment by gurezo** [comment]
@hmatrjp @potappo 
CHIRIMEN コミュニティは、参加しないことになったので、MDN 翻訳コミュニティブースの設営やブース案内終日出来る見込みになりました。

**Reply by potappo** [reply]
@gurezo 了解です。当日のブース番の時間帯については、近くなったら考えましょう。

**Comment by gurezo** [comment]
@potappo 
状況を教えてください。
すっかりお任せしていたのですが、OSC の申込みは完了下という事で良いでしょうか？

**Reply by potappo** [reply]
@gurezo 
忙しくて報告忘れていましたが、申し込みは完了しています。
https://event.ospn.jp/osc2023-fall/exhibit

**Reply by gurezo** [reply]
@potappo 
申し込み有難うございました！
開催日時に近くなったら、相談しましょう！

**Comment by potappo** [comment]
@hmatrjp @gurezo 
ブースで配るノベルティグッズをどうしましょう？の相談です。

送り先住所は3日前に判明していたのですが、今月に入ってずっと休みがないのでこのタイミングになりました。

今回は会場に直接送れるので（送り先は必要な方に Slack の DM などでお伝えします）、
前回と同じサイズの箱で濱崎さんが直接会場へ送るという方法でどうでしょうか？前回余った分は、私が当日持ち込みます。
なるべく配り切ろうと思いますが、当日のブース出展担当者で持ち帰れないほど余った場合は、着払いで送り返します。

改善点など、ご意見あればお願いします。

**Reply by gurezo** [reply]
@potappo cc @hmatrjp 
提案内容で良いとともいます！
念のため、今週末のミートアップで最後のツメを相談できれば、問題ないと思いました。

**Reply by hmatrjp** [reply]
会場に直接送れるのはいいですね。
箱はありものの流用です、今あるのは、22 x 28 x 40 (100サイズ)ですが、加工して高さを縮めることはできます。
大きすぎるようなら、スーパーかどこかで空き箱を見繕います。
持ち帰りきれない余りの着払いの返送もOKです。

何をいくつ、は最終的には箱の大きさで決まりますが、これを優先したい、という品目のリクエストがあれば、先に入れます。

**Reply by hmatrjp** [reply]
さきほど倉庫に行って、送る物を仮に選んで持ってきました。箱は、60サイズ(17x17x24)に変えました。

1. ステッカー All Hands 2010 Buddy Quest (15)
2. ステッカー All Hands 2017 (10)
3. ノート無地 (5)
4. ノート横罫、ボールペン付き(10) ボールペンは出るまで長く転がす必要のあるものもある
5. スタイラス付きボールペン (10)
6. ストラップ (10)
7. 充電ケーブル (10)

前回と比べてステッカーが少なめですが、3 以降の品の方が Mozilla のグッズだとわかりやすいかもしれません。

**Reply by hmatrjp** [reply]
![image](https://github.com/mozilla-japan/translation/assets/2672068/210dab8e-9d02-458c-bac9-1c0c35584a10)

**Reply by potappo** [reply]
今回余ったグッズは月曜午後着で私の自宅に送りました。残数はそのときに報告します！

**Reply by potappo** [reply]
以下、グッズ残数です。増えているようにみえるのは、前回の残りも含まれているためです。
ステッカー All Hands 2010 Buddy Quest 11枚
ステッカー All Hands 2017 6枚
ノート横罫、ボールペン付き 6冊
ノート無地  8冊
スタイラス付きボールペン 4本
ストラップ 1本
充電ケーブル  1本

**Reply by hmatrjp** [reply]
次回以降の準備する数の参考になればと、こちらのシートに、残数と配布数を計算して記入してみました。
https://docs.google.com/spreadsheets/d/1N73mciqvlQbuC4457OMteS-TXDldj75XybDKfK8CToI/edit#gid=345022493
台湾コミュニティーに追加を依頼する目安にもなるかと思うので、もし、他で配ったりして減っているとか、間違いにお気づきの点ありましたら
修正をお願いします。 @potappo @gurezo 

それから、秋の方の、ブース来場者数はわかりますか？　（32行目）　おおよそで構いません。

**Reply by gurezo** [reply]
@hmatrjp cc @potappo 
懇親会前に主催者の宮原さん発表によると３００人に達したとのことです。
詳細な人数は、OSC事務局の発表街です。

**Reply by hmatrjp** [reply]
ありがとうございます。 @gurezo 

ということは、全体来場人数で比較すると、秋が春の3分の2ですね。
グッズの出具合からすると、ブース来場者数も、だいたい比例すると考えても良いのかも。

**Reply by gurezo** [reply]
@hmatrjp 

> グッズの出具合からすると、ブース来場者数も、だいたい比例すると考えても良いのかも。

はい。その通りだと思います。

**Comment by elie-j** [comment]
@potappo @gurezo 
すみません！ちょっと立て込んでいて、こちらを全然ウォッチできていなかったのですが、来週末(10/21)OSCの展示出展される予定だったのですね。
何かWebDINOから貸し出しが必要なものとかあれば、10/16(月)までに送付先の宛先や住所、電話番号と一緒に必要なものを教えてもらえれば、火曜あたりであればオフィスに寄れそうなので対応できると思いますのでお知らせください m(_ _)m

ノートPCは、確認していないのでわからないのですが、たぶん前回お貸しして戻ってきてから使っていない可能性が高いのでけっこうアプデが走るかもです。

**Reply by potappo** [reply]
@elie-j 
ありがとうございます。
明日14日の翻訳ミートアップで相談して必要なものがあれば依頼させていただきます。

**Comment by gurezo** [comment]
本日のブース
![image](https://github.com/mozilla-japan/translation/assets/7631567/c3d68b50-f016-430c-a811-6540cf14f6df)
![image](https://github.com/mozilla-japan/translation/assets/7631567/c410302b-2141-41a6-9feb-9b0534005119)
![image](https://github.com/mozilla-japan/translation/assets/7631567/ed1d7e05-4f8e-4609-95ba-1e8ccdc77331)
![image](https://github.com/mozilla-japan/translation/assets/7631567/1b80cdf8-500c-423c-bf72-8f2e5fa24e6f)
![image](https://github.com/mozilla-japan/translation/assets/7631567/c7252b2c-4ae9-48dd-91bd-96b941629952)

**Comment by potappo** [comment]
OSC 2023 東京秋へのブース出展は無事に終了しました。お疲れ様でした！今回の出展覚書です。

- 翻訳イベントのリアル開催へのニーズはあるので、どこかでやりたい
- コミュニティ紹介の LT をする？
- 翻訳ガイドを技術同人誌化する？
- W3C の方が来たので、何かしら連携できればしたい
-  Pio がある東口方面はコンビニしかない。京急蒲田西口方面はラーメン屋が多い。ランチタイムは11時半～。
- 次回の東京春は 3/10（日） 浅草。こちらは同じ建物内に食堂があります。

反省点は以下でした。
- ステッカーがノートの下に埋もれていて、気付くまで配れていませんでした。次回はグッズが一式展開できているか確認します。
- Twitter に上手くログインできなかったので、次回は事前にログインしておきます、

**Comment by potappo** [comment]
2024 春の分を立てましたので、こちらはクローズします。
